packages/client/src/comp/states/Base.tsx

```diff
@@ -234,13 +235,12 @@ const ContainerlessBase = <Info extends StateInfo>({ enrichValues, children }: P
     <Togglable
       label='Mail my ballot to a different address than listed above'
     >{
-      (checked) => <BaseInput
-        id='mailing'
-        value={fields.mailing.value}
-        label='Mailing Address'
-        required={checked}
-        onChange={e => updateField('mailing', e.currentTarget.value)}
-      />
+      (checked) => checked && (
```

Comment:
You don't need `checked &&` -- `checked` is only used to indicate that fields are required